docs/src/getting-started/beginner-guide.rst

```diff
@@ -0,0 +1,279 @@
+  model is learning well or overfitting. This is like giving a student quiz problems to
+  check their understanding.
+
+- **Test set** (typically 10-15% of data): Used **only at the end** to evaluate final
+  performance. This is like the final exam - the student hasn't seen these problems during
+  study.
+
+**Why split the data?** If we only used a training set, the model might just "memorize"
+the examples rather than learning general patterns. The validation and test sets help us
+verify that the model has learned generalizable knowledge.
+
+Epochs and Training Iterations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Epoch**: One complete pass through all training data. If you have 1000 structures and
+  train for 100 epochs, the model sees each structure 100 times.
+
+- **Batch size**: How many structures the model processes at once. For example, with 1000
+  structures and a batch size of 32, each epoch involves ~31 steps (batches).
+
+**How many epochs do I need?** Typically 50-200 epochs for most systems. Too few, and the
+model hasn't learned enough. Too many, and it might "overfit" (memorize training data
+instead of learning patterns).
+
+Overfitting vs Underfitting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Underfitting**: The model is too simple and hasn't learned enough. Like trying to fit
+  a curve through complex data with just a straight line.
+
+  *Signs*: High errors on both training and validation sets.
+
+  *Solution*: Train longer (more epochs) or use a more complex model.
+
+- **Overfitting**: The model has memorized the training data but doesn't generalize. Like
+  memorizing answers to practice problems without understanding the concepts.
+
+  *Signs*: Low training error but high validation error.
+
+  *Solution*: Train for fewer epochs, use more training data, or simplify the model.
+
+Loss Functions
+^^^^^^^^^^^^^^
+
+The **loss function** measures how wrong the model's predictions are. During training, the
+model adjusts itself to minimize this loss.
+
+For atomistic systems, common losses include:
+
+- **Energy loss**: Difference between predicted and true energies
+- **Force loss**: Difference between predicted and true forces (derivatives of energy)
+- **Stress loss**: Difference between predicted and true stress (for periodic systems)
+
+You can train on multiple properties simultaneously by combining their losses with
+different weights.
+
+Learning Rate
+^^^^^^^^^^^^^
+
+The **learning rate** controls how much the model adjusts itself based on errors. Think of
+it like step size when searching for the bottom of a valley:
+
+- **Too large**: The model jumps around and might miss the best solution
+- **Too small**: Training is very slow
+- **Just right**: The model efficiently finds a good solution
+
+Most architectures in metatrain use adaptive learning rates that automatically adjust
+during training, so you usually don't need to worry about this.
+
+Model Architectures
+-------------------
```

Comment:
We will forget about updating this list. Also it draws attention to specific models and I don't if we want this.